Tasks/PublishPipelineArtifactV1/icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
@willsmythe - Does the image of V0 need to change as well??

Reply:
I have asked Will. No, we are not going to change the icons for old tasks.